commands/operator-sdk/cmd/add/api.go

```diff
@@ -71,114 +70,43 @@ Example:
 }
 
 func apiRun(cmd *cobra.Command, args []string) {
-	projectPath := cmdutil.MustInProjectRoot()
-	fullProjectPath := mustGetwd()
-
 	// Create and validate new resource
 	r, err := scaffold.NewResource(apiVersion, kind)
 	if err != nil {
 		log.Fatal(err)
 	}
 
-	// Must be new Kind: pkg/apis/<group>/<version>/<kind>_types.go shouldn't exist
```

Comment:
@estroz Do you know what happens if you try the command add api twice?
I expect it to fail for the same apiversion and kind. But for a different Kind I expect it to succeed:
```sh
$ operator-sdk add api --api-version=app.example.com/v1alpha1 --kind=AppService
Create /Users/haseeb/work/go-space/src/github.com/example-inc/app-operator/pkg/apis/addtoscheme_app_v1alpha1.go
Create /Users/haseeb/work/go-space/src/github.com/example-inc/app-operator/pkg/apis/app/v1alpha1/appservice_types.go
Create /Users/haseeb/work/go-space/src/github.com/example-inc/app-operator/pkg/apis/app/v1alpha1/register.go
Create /Users/haseeb/work/go-space/src/github.com/example-inc/app-operator/pkg/apis/app/v1alpha1/doc.go
Create /Users/haseeb/work/go-space/src/github.com/example-inc/app-operator/deploy/crds/app_v1alpha1_appservice_crd.yaml
Create /Users/haseeb/work/go-space/src/github.com/example-inc/app-operator/deploy/crds/app_v1alpha1_appservice_cr.yaml

# Add api for an existing kind(type)
$ operator-sdk add api --api-version=app.example.com/v1alpha1 --kind=AppService
2018/10/09 12:16:21 /Users/haseeb/work/go-space/src/github.com/example-inc/app-operator/pkg/apis/app/v1alpha1/appservice_types.go already exists

# Add a new kind
$ operator-sdk add api --api-version=app.example.com/v1alpha1 --kind=Memcached
Create /Users/haseeb/work/go-space/src/github.com/example-inc/app-operator/pkg/apis/addtoscheme_app_v1alpha1.go
Create /Users/haseeb/work/go-space/src/github.com/example-inc/app-operator/pkg/apis/app/v1alpha1/memcached_types.go
Create /Users/haseeb/work/go-space/src/github.com/example-inc/app-operator/pkg/apis/app/v1alpha1/register.go
Create /Users/haseeb/work/go-space/src/github.com/example-inc/app-operator/pkg/apis/app/v1alpha1/doc.go
Create /Users/haseeb/work/go-space/src/github.com/example-inc/app-operator/deploy/crds/app_v1alpha1_memcached_crd.yaml
Create /Users/haseeb/work/go-space/src/github.com/example-inc/app-operator/deploy/crds/app_v1alpha1_memcached_cr.yaml
```
```sh
$ tree pkg/apis/
pkg/apis/
├── addtoscheme_app_v1alpha1.go
├── apis.go
└── app
    └── v1alpha1
        ├── appservice_types.go
        ├── doc.go
        ├── memcached_types.go
        ├── register.go
        └── zz_generated.deepcopy.go
```

Reply:
As of this most recent commit, this is the encountered behaviour on my machine.